templates/index.html

```diff
@@ -57,8 +57,14 @@ <h3 class="fs-title">
 </div>
 
 {% if CASS_HOME_EXTRAS %}
-    {% set home_extras = pages[7] %}  <!-- Set to 6 to grab homepage extras --> 
+    {% set home_extras = pages[7] %}  <!-- Set to 7 to grab homepage extras --> 
     {{ home_extras.content }}
 {% endif %}
 
+{% if OSL_HOME_EXTRAS %}
+    {% set home_extras = pages[4] %}  <!-- Set to 4 to grab homepage extras --> 
```

Comment:
Pelican generates a page for each article and the ``about.rst`` happens to be the fifth element in the list of pages. It's unclear to me how page order is determined.